egs2/TEMPLATE/asr1/pyscripts/audio/format_wav_scp.py

```diff
@@ -290,7 +290,7 @@ def generator():
     with out_num_samples.open("w") as fnum_samples:
         for uttid, (wave, rate), wavpath, subtypes in tqdm(generator()):
             save_asis = True
-            if args.fs is not None and args.fs != rate:
+            if args.fs is not None and args.fs != rate and len(wave) != 0:
```

Comment:
Do you get some audio with zero length? If so maybe raise a warning as well.

Reply:
I think I ran into this in acesinger. I will double-check and add a warning.